src/routes.js

```diff
@@ -185,7 +186,11 @@ const routes = [
   {
     path: '/',
     component: App,
-    routes: defaultRoutes,
+    routes: [
+      // addon routes have a higher priority then default routes
+      ...(addonRoutes || []),
```

Comment:
If so, they should go the last, right?

Reply:
No, the router use the first match in the list